scripts/openapi/gong/metadata/main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"log/slog"
+
+	"github.com/amp-labs/connectors/providers/gong/metadata"
+	"github.com/amp-labs/connectors/providers/gong/openapi"
+	"github.com/amp-labs/connectors/tools/fileconv/api3"
+	"github.com/amp-labs/connectors/tools/scrapper"
+	"github.com/getkin/kin-openapi/openapi3"
+)
+
+var ignoreEndpoints = []string{ // nolint:gochecknoglobals
+	"/v2/settings/scorecards",
+	"/v2/settings/trackers",
+	"/v2/stats/activity/scorecards",
+	"/v2/data-privacy/data-for-email-address",
+	"/v2/data-privacy/data-for-phone-number",
+	"/v2/crm/entities",
+	"/v2/crm/entity-schema",
+	"/v2/crm/request-status",
+	"/v2/library/folder-content",
+	"/v2/calls/manual-crm-associations",
+	"/v2/crm/integrations",
+	"/v2/permission-profile", // GET, single item
+	"/v2/permission-profile/users",
+	"/v2/users/extensive",
+	"/v2/library/folders",
+	//
+	// Endpoints that require query parameters.
+	//
+	// Requires query parameter: workspaceId.
+	// Response field is `profiles`.
+	"/v2/all-permission-profiles",
+	// Requires query parameters: workspace-id, manager-id, from, to.
+	// Response field is `coachingData`.
```

Comment:
Don't we have a function that maps objects to the response field now? Correct me if I am wrong

Reply:
This line is a bit confusing. I will be removing alike. The reason why URL is ignored is because it requires query parameters, thats the primary reason. The response field name is just extra info, trully redundant and creates confusing of why that is mentioned.